OpenRA.Mods.Common/Traits/GainsExperience.cs

```diff
@@ -33,6 +33,9 @@ public class GainsExperienceInfo : ITraitInfo, Requires<ValuedInfo>
 		[Desc("Palette for the level up sprite.")]
 		[PaletteReference] public readonly string LevelUpPalette = "effect";
 
+		[Desc("Experience factor for each levelup. Will default to Valued Cost.")]
+		public readonly int RequiredExperience = -1;
```

Comment:
Considering #15529, it might be a better option to have an enum defined here to choose between absolute values for the levels relative values to a base value and relative values to Valued->Cost.

Reply:
I think this approach is ok. `RequiredExperience: 1` would give absolute value. Tho maybe ExperienceFactor would be a better name to indicate that it is a multiplier.

Reply:
Alright then.

Reply:
@IceReaper: Can we please address these comments by renaming this to `ExperienceModifier` (for consistency with `DamageMultiplier`, `FirepowerMultiplier`, etc) and the description to "Multiplier to apply to the Conditions keys.  Defaults to the actor's value."  You will also need to change the description for `Conditions` above.